server/server.js

```diff
@@ -218,15 +204,15 @@ app.post('/delete/:id', (req, res) => {
     return;
   }
 
-  storage
-    .delete(id, delete_token)
-    .then(err => {
-      if (!err) {
-        log.info('Deleted:', id);
-        res.sendStatus(200);
-      }
-    })
-    .catch(err => res.sendStatus(404));
+  try {
+   const err = await storage.delete(id, delete_token);
```

Comment:
Super minor nit, but I think the indenting is slightly off here. Not sure if you want to try risking running `npm run format` and running the **prettier** formatter against the codebase.